index.html

```diff
@@ -50,6 +50,108 @@ <h5>LET'S CONNECT</h5>
+               <p>CANOPY</p>
+          <div class="dot"></div>
+               <p>Back End Dev</p>
+          <div class="dot"></div>
+               <p>2015</p>
+          </div>
+            </div>
+      </div>
+    </div>
+           <p class="information">A daily selection of privately  personalized  reads; no accounts or sign-ups required.</p>
+     
+         <ul class="buttons">
+             <li><a href=""><img src="./images/Tag (3).svg" alt=""></a></li>
+             <li><a href=""><img src="./images/Tag (4).svg" alt=""></a></li>
+             <li><a href=""><img src="./images/Tag (5).svg"alt=""</a></li>
+         </ul>
+</Section>
+
+<section class="wrapper">
+  <div class="top">
+      <h1>About
+          <br>Myself</h1>
+      <p class="normaltxt">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+      <div class="social">
+          <h3>LET'S CONNECT</h3>
+
+          <ul class="vectorfram">
+              <li>
+            <a href="#"><button class="vector-button" type="button"><img src="./images/github.svg" alt="github"><button></button></a>
+              </li>
+              <li>
+                  <a href="#">button class="vector-button" type="button"><img src="./images/linkedin.svg" alt="linked in"></button></a>
+              </li>
+              <li>
+                  <<a href="#"><button class="vector-button" type="button"><img src="./images/Vector.svg" alt="vector"></button></a>
+              </li>
+              <li>
+                <a href="#"><button class="vector-button" type="button"><img src="./images/twitter.svg" alt="twitter"></button></a>
+              </li>
+              <li>
+                <a href="#"><button class="vector-button" type="button"><img src="./images/medium.svg" alt="m"></button>
+              </a></li>
+          </ul>
+
+      </div>
+
+
+      <button type="button" class=" button">
+              Get my resume
+          </button>
+  </div>
+
+  <div class="skill ">
+      <div class="listfram ">
+          <div class="listtag ">
+              <p class="listname">Languages</p>
+              <button type="button" class="icon"><img src="./images/medium.svg" alt="button"></button>
+          </div>
+          <ul class="skillul ">
+              <li class="skillli ">
+                  <div class="listimg"><img src="./images/linkedin.svg" alt="javascript"></div>
+                  <p class="listtxt">JavaSript</p>
+              </li>
+              <li class="skillli ">
+                  <div class="listimg"><img src="./images/github.svg" alt="html"></div>
+                  <p class="listtxt">HTML</p>
+              </li>
+              <li class="skillli ">
+                  <div class="listimg"><img src="./images/cssaboutme.svg" alt="CSS"></div>
+                  <p class="listtxt">CSS</p>
```

Comment:
Kindly make sure all icons displays as seen in the original design.
Your Design  |  Figma Design
![screenshot (2)](https://user-images.githubusercontent.com/44624138/170845586-972749f1-0262-49c7-a726-9d4a92e73af1.png)  |  ![screenshot (5)](https://user-images.githubusercontent.com/44624138/170845592-c2cdc875-93de-43ea-9258-649cb917fbeb.png)

Reply:
Please make sure the sections uses the font and colours (background and font) defined for the template. 👍🏽